resources/packaging/windows/scripts/npcap.ps1

```diff
@@ -0,0 +1,25 @@
+[CmdletBinding()]
+param (
+	[Parameter(Mandatory)]
+	[ValidateSet("amd64", "i386", "arm64")]
+	[string]$ARCH,
+	[Parameter(Mandatory)]
+	[string]$OEM
+)
+
+Write-Host "::group::Npcap SDK"
+Invoke-WebRequest -Uri "https://npcap.com/dist/npcap-sdk-1.13.zip" -OutFile "$env:TEMP\\npcap-sdk.zip" -Verbose
+Expand-Archive -LiteralPath "$env:TEMP\\npcap-sdk.zip" -DestinationPath "$env:TEMP\\npcap-sdk" -Verbose
+$LibPath = switch ($ARCH)
+{
+	"amd64" { "Lib\\x64"   }
+	"i386"  { "Lib"        }
+	"arm64" { "Lib\\ARM64" }
```

Comment:
I initially had put it in a separate file since I had to call the script for both building and installation, but we excluded the installer usage later on. Besides, I kept all possible architectures supported by _Npcap_ just in case we want to build more platforms in the future. I will make adjustments here as well ;)

Reply:
Perfect, after this one I think we are almost done.
I'll only reach out to you via e-mail to update about the tests I've performed and the missing ones 👍